tests/test_fields.py

```diff
@@ -25,6 +28,15 @@ def __init__(self, file=None, created=None):
         self.created = created or datetime.datetime.now()
 
 
+class DownloadableBase64Image(object):
+    class ImageFieldFile(object):
```

Comment:
When using models.ImageField in a subclass of models.Model, then
the class that the serializer gets access to is ImageFieldFile which has a path
attribute. From what I’ve tried, to test this would require setting up the model and a db 
so I just simulate it here.